RecoParticleFlow/PFTracking/interface/PFTrackAlgoTools.h

```diff
@@ -18,5 +18,6 @@ namespace PFTrackAlgoTools {
                         const std::vector<double>& DPtovPtCut,
                         const std::vector<unsigned>& NHitCut,
                         bool useIterTracking,
+                        const reco::TrackBase::TrackQuality trackQuality = reco::TrackBase::highPurity,
```

Comment:
I checked, that various inputs in the configuration are correctly converted into the proper enums.
I can remove the default value, since I included the default value in the input of the two other plugins making use of this function. Hence for the 3 plugins I adjusted, there should be no difference wrt the previous version.